### ISSUE TITLE
Quartzite exception on master

### ISSUE THREAD

**Issue by FallenChromium** [metabase/metabase#9165]
Hello! So, the title says pretty much everything, I'm getting `EOFException` either on 0.31.2 release or master branch, as of now I didn't digged much into reasons of such behavior. Here's details:
- Your browser and the version: Safari 12.0.2
- Your operating system: macOS 10.14.2 Mojave
- Your databases: PostgreSQL 11.1
- Metabase version: 0.31.2, 0.32.0-snapshot (master) EDIT: also on 1.0.3 (pulled from docker just for fun)
- Metabase hosting environment: Linux/Ubuntu 18.04
- Metabase internal database: PostgreSQL

*Repeatable steps to reproduce the issue*
Um... It happens without exact pattern, I just seeing this occasionally in journalctl, so as for now - no details.
Log:
```
Couldn't retrieve trigger: null [See nested exception: java.io.EOFException]
at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1524)
at org.quartz.impl.jdbcjobstore.JobStoreSupport.recoverMisfiredJobs(JobStoreSupport.java:968)
at org.quartz.impl.jdbcjobstore.JobStoreSupport.doRecoverMisfires(JobStoreSupport.java:3174)
at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.manage(JobStoreSupport.java:3934)
at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.run(JobStoreSupport.java:3955)
Caused by: java.io.EOFException
 at java.base/java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2756)
at java.base/java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:3251)
at java.base/java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:866)
at java.base/java.io.ObjectInputStream.<init>(ObjectInputStream.java:342)
at org.quartz.impl.jdbcjobstore.PostgreSQLDelegate.getObjectFromBlob(PostgreSQLDelegate.java:101)
at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectTrigger(StdJDBCDelegate.java:1764)
at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1520)
```
P.S can't wait for more informative error details in Clojure 1.10 😆

**Comment by camsaul** [comment]
I don't think Clojure 1.10 is going to make any more difference here. Is this causing any bugs other than the message?

**Comment by camsaul** [comment]
This should be fixed in 0.32.x. Are you still seeing issues @FallenChromium ?

**Comment by FallenChromium** [comment]
Yeah, I don't see this in logs now. Thanks!
